harvester/civic.py

```diff
@@ -96,9 +96,17 @@ def convert(gene_data):
                 association = el.evidence_label(
                     evidence_item['evidence_level'], association, na=True
                 )
-                association = ed.evidence_direction(
-                    evidence_item['clinical_significance'], association
-                )
+
+                description = evidence_item['clinical_significance'].lower()
+                if description == 'positive':
+                    description = 'positive outcome'
+                if description == 'negative':
+                    description = 'negative outcome'
+
+                association['response_type'] = '{} {} {}'.format(
+                    evidence_item['evidence_direction'],
+                    evidence_item['evidence_type'],
+                    description)
```

Comment:
add check if `description == 'n/a'`. If so, do not add `description` to `response_type` string.

otherwise, CIViC looks good!

Reply:
civic

### before
![image](https://user-images.githubusercontent.com/47808/42353317-93cfce58-8074-11e8-8925-ca0a8529a4cc.png)

### after 
![image](https://user-images.githubusercontent.com/47808/42353320-9c28587c-8074-11e8-97b5-b3d55c8f3d0c.png)